contracts/contracts/coordination/SigningCoordinator.sol

```diff
@@ -46,6 +53,12 @@ contract SigningCoordinator is Initializable, AccessControlDefaultAdminRulesUpgr
         uint256[20] gap;
     }
 
+    struct Server {
```

Comment:
I think we should find a more generic term for `Server`, which is too specific to a particular use case. What about something like `Domain`?